src/scripts/main.coffee

```diff
@@ -119,7 +146,21 @@ class FormbuilderCollection extends Backbone.Collection
       correctType = model.get('type') in ['dropdown', 'checkbox', 'radio', 'approval']
       differentModel = model != child
       hasNoParent = !model.hasParent()
-      correctType and differentModel and hasNoParent
+      flag = true
+      uuid = child.attributes.uuid
+      uuid_parent
+      ascent
+      parent = model.conditionalParent()
+      if parent
+        ascent = model.findConditionalParent()
+        uuid_parent = parent.attributes.uuid
+        if ascent
+          if ascent.lenth > 0
```

Comment:
Assuming this should be `length`. And can these be grouped together into one `ascent && ascent.length>0`?